provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -55,22 +55,21 @@ def generate(self) -> str:
                         stderr=S.PIPE,
                         encoding='utf8')
 
+        dhall_out, err_d = dhall.communicate(input=gencdf)
```

Comment:
@SwapnilGaonkar7 What problem was observed and what is the root cause?

Reply:
BTW perhaps the issue can be caused by the following: we never invoke `dhall.stdout.close()` but it seems like we should. Please take a look at this SO: https://stackoverflow.com/questions/6780035/how-to-run-ps-cax-grep-something-in-python

Reply:
As concluded in the chat with @SwapnilGaonkar7, we'll merge this hotfix because it can unblock other teams quickly and in the meantime another PR with a deeper analysis and proper fix will be provided.